app/src/main/res/layout/fragment_login_screen.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/backgroundColor"
+    tools:context=".login_screen.LoginScreen" >
+
+    <TextView
```

Comment:
I think it would be much better if you set the screen title in Toolbar as google material design says

Reply:
Done